website/docs/getting-started/importing-saved-containers.md

```diff
@@ -0,0 +1,44 @@
+---
+id: importing-saved-containers
+title: Importing saved containers
+description: Import saved containers to migrate transparently to Podman, and continue using familiar containers.
+tags: [podman-desktop, getting-started, importing]
+keywords: [podman desktop, podman, containers, importing]
+---
+
+# Importing saved containers to Podman
+
+Consider importing saved containers to continue using familiar containers.
+
+#### Prerequisites
+
+* [Podman](../Installation/macos-install)
```

Comment:
hello, out of curiosity, why are we referring to a macOS installation ?

Reply:
Removing the link for the moment. We miss an aggregating section.

```suggestion
* Podman
```